projects/plugins/jetpack/extensions/blocks/sharing-button/sharing-button.php

```diff
@@ -65,7 +65,8 @@ function render_block( $attr, $content, $block ) {
 	Jetpack_Gutenberg::load_assets_as_required( __DIR__ );
 
 	$component  = '<li class="jetpack-sharing-button__list-item">';
-	$component .= '<a rel="nofollow noopener noreferrer" class="' . $sharing_link_class . '" href="' . $link_url . '" target="_blank" data-shared="' . $data_shared . '" aria-label="' . $link_aria_label . '" primary>';
+	$component .= '<a rel="nofollow noopener noreferrer" class="' . $sharing_link_class . '" href="' . $link_url . '" target="_blank" ';
```

Comment:
We could escape attributes here to be on the safer side, but maybe that's me being over-cautious and now fully aware of the data flow here.

Reply:
Thanks! Thats a good point. Do you suggest using `htmlspecialchars` here for params?

Reply:
`esc_attr` for attributes, including `aria-label`.